sources/documents/2_RTB/PianoDiProgetto/Pianificazione/Sprint/sprint-7.tex

```diff
@@ -0,0 +1,18 @@
+\subsubsection{Sprint 7: da 2024-06-25 a 2024-07-02}
+Il team in vista della RTB si appresta a concludere le attività di programmazione relative al \glossario{PoC} e nell'ultimare le sezioni incomplete dei documenti, in modo da porre maggiore prima della consegna alla sezione documentale.
```

Comment:
```suggestion
Il team in vista della RTB si appresta a concludere le attività di programmazione relative al \glossario{PoC} e nell'ultimare le sezioni incomplete dei documenti, focalizzandosi maggiormente sulla sezione documentale  prima della consegna.
```
Non riesco a capire, forse intendevi dire questo?

Reply:
si era "in modo di porre maggiore attenzione ...", ma si è perso nella mia testa. Meglio la tua